sci-misc/elmer-fem/elmer-fem-9.0-r1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Could we also bump this to `EAPI=8` while we're touching this ebuild anyway.

Reply:
I've amended the commit and can confirm bumping to EAPI=8 doesn't break anything here.